tools/lambda-promtail/lambda-promtail/main.go

```diff
@@ -47,6 +53,21 @@ func init() {
 	username = os.Getenv("USERNAME")
 	password = os.Getenv("PASSWORD")
 
+	extraLabelsRaw = os.Getenv("EXTRA_LABELS")
+	extraLabelsSplit := strings.Split(extraLabelsRaw, ",")
+	if len(extraLabelsRaw) > 0 {
```

Comment:
Thanks for the change. Can you also add small test to lock this behavior? (prefer to move it to different function `parseLabels()` or similar and have some tests cases for)

I can merge after that.